integreat_compass/cms/templates/index.html

```diff
@@ -4,15 +4,114 @@
     {% translate "Home" %}
 {% endblock title %}
 {% block content %}
-    <div class="absolute top-0 right-0 bottom-0 left-0 flex justify-center items-center bg-background-900 z-1">
-        <div class="flex flex-col justify-center gap-4 text-center">
-            <h1 class="font-bold text-5xl drop-shadow-md mb-12 text-white">{% translate "Arriving, reimagined." %}</h1>
-            <a href="#"
-               class="bg-primary hover:bg-red-500 text-white font-bold text-3xl py-4 px-4 rounded-2xl">{% translate "Discover Integreat Compass" %}</a>
-            <a href="{% url 'cms:protected:new_offer' %}"
-               class="bg-white hover:bg-gray-200 font-bold text-3xl py-4 px-4 rounded-2xl">{% translate "Add new offer" %}</a>
+    <div class="flex container mx-auto h-[calc(100vh-6rem)]">
+        <div class="hidden lg:block fixed top-24 bottom-0 bg-white lg:w-3/12 py-8 pr-8 shadow-[10px_0_5px_-2px_rgba(0,0,0,0.1)]">
+            <form action="{% url 'cms:public:index' %}" method="post">
```

Comment:
One way I was able to make it work was by playing with fire and adding negative margin to the sidebar on the left and adding the same spacing as (positive) padding to the form. With this the spacing evens out and we have the expected result 
![grafik](https://github.com/digitalfabrik/integreat-compass/assets/72705147/8a6224af-d040-463d-a1c9-ca18b1d29e09)
However this is not a great solution since the code is really messy (3xl:ml-[calc(-1*(calc(calc(100vw-1200px)/2)))] lg:ml-[calc(-1*(calc(calc(100vw-1024px)/2)))]) and for the form element (3xl:ml-[calc(calc(100vw-1200px)/2)] lg:ml-[calc(calc(100vw-1024px)/2)]) and still there is no guarantee that it will work for every width (especially ultra large screens will look awful). 

> We could just remove the left margin for this view

I think we would need to remove the mx-auto from the base view and then instead add it to every template that needs it. To me that is also not a great solution. Does someone maybe have another idea?

Reply:
> Does someone maybe have another idea?

Instead of `<div class="container">`, you could write `<div class="{% block container_width %}max-w-... md:max-w-... etc.{% endblock container_width %}">` in the base template, and then in the `offer-detail-layover.html` template, you could overwrite this by adding `{% block container_width %}max-w-... md:max-w-... etc.{% endblock container_width %}` with the different width styles.

Reply:
Thank you for your suggestion. I took the main part, but changed the details a bit, I hope the container works as espected